playbook/roles/devtools/defaults/main.yml

```diff
@@ -20,3 +20,8 @@ xdebug:
 
 # this can be set to true to add the remote log file.
 xdebug_remote_debug_log: false
+
+enable_nodejs: true
+nodejs_version: v9.4.0
```

Comment:
I would be in favour of using the 8.* LTS release. If we have a version mismatch between local environments and the deployment context (like deploybot) I would rather have the older version locally.